doc/release_notes.rst

```diff
@@ -10,6 +10,8 @@ Release Notes
 PyPSA-WAL Upcoming Release
 ==========================
 
```

Comment:
New feature: CCL additions can be now customized using data/walloon/custom_constraints